index.html

```diff
@@ -62,6 +62,7 @@
                         <div id="navbar" class="navbar-collapse collapse navbar-right">
                             <ul class="nav navbar-nav">
                                 <li class="active"><a href="{{ site.baseurl }}/">Home</a></li>
+                                <li><a href="{{ site.baseurl }}/FAQ/">FAQ <span class="badge">BETA</span></a></li>
```

Comment:
That is strange. I added it here.

Reply:
Sorry, my bad, was a caching issue.

Reply:
I can see it when I open the website. I guess this is the browser cache...